src/test/java/pages/HomePage.java

```diff
@@ -1,20 +1,47 @@
 package pages;
 
 import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
 
 public class HomePage extends BasePage {
 
     By userAvatarIcon = By.cssSelector("img.avatar");
 
+    By firstPlaylist = By.cssSelector(".playlist:nth-child(3)");
+
+    By playlistInputField = By.cssSelector("input[name='name']");
```

Comment:
I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");